vendor/github.com/nishanths/fullstory/client.go

```diff
@@ -65,11 +66,17 @@ func (c *Client) doReq(req *http.Request) (io.ReadCloser, error) {
 		defer resp.Body.Close()
 		b := &bytes.Buffer{}
 		io.Copy(b, resp.Body) // Ignore error.
-		return nil, StatusError{
+		statusError := StatusError {
 			Body:       b,
 			Status:     resp.Status,
 			StatusCode: resp.StatusCode,
 		}
+
+		if resp.StatusCode == http.StatusTooManyRequests {
+			statusError.RetryAfter = resp.Header.Get("Retry-After")
```

Comment:
`Retry-After` header can be set even if it is not a `http.StatusTooManyRequests`. I believe status `503` may also come with a `Retry-After` header. So you can probably just do this:
```go
return nil, StatusError{
  Body:       b,
  Status:     resp.Status,
  RetryAfter: resp.Header.Get("Retry-After"),
  StatusCode: resp.StatusCode,
}